.github/workflows/release.yml

```diff
@@ -1,59 +1,68 @@
 name: LPM Installer Publish
-run-name: LPM Installer Publish     
+run-name: Publish LPM Installer ${{ github.ref_name }}
+
 on:
   push:
     tags:
-      - '*'
+      - v*
+
 jobs:
-  Explore-GitHub-Actions:
-    runs-on: windows-latest
+  release-installer:
+    runs-on: windows-2022
+    timeout-minutes: 2
+    env:
+      SM_API_KEY: ${{ secrets.SM_API_KEY }}
+      SM_CLIENT_CERT_FILE: "D:\\Certificate_pkcs12.p12"
+      SM_CLIENT_CERT_PASSWORD: ${{ secrets.SM_CLIENT_CERT_PASSWORD }}
+      SM_HOST: ${{ secrets.SM_HOST }}
```

Comment:
Ideally these variables should be defined only for the step that requires them but I'm not sure which one requires which variables. If you help me with that I'll move `env` block from job level to step level.

Reply:
I don't know exactly which env variables pertain to which step, as this code was brought in from a digicert sample. I think for the sake of time right now I prefer to keep them grouped together upfront.

Reply:
This is precisely why it's better to have them defined for each step. Alright, maybe we'll improve it the next time.

BTW have you checked the workflow after the merge?